guides/release/routing/defining-your-routes.md

```diff
@@ -65,7 +65,7 @@ Router.map(function() {
 ```
 
 The route defined above will by default use the `blog-post.js` route handler,
-the `blog-post.hbs` template, and be referred to as `blog-post` in any
+the `blog-post.gjs` template, and be referred to as `blog-post` in any
```

Comment:
Above here (I cannot comment on unedited lines) are references to `template` and examples of hbs that need to be converted.